wled00/colors.cpp

```diff
@@ -268,7 +268,11 @@ void loadCustomPalettes() {
               uint8_t rgbw[] = {0,0,0,0};
               if (colorFromHexString(rgbw, pal[i+1].as<const char *>())) { // will catch non-string entires
                 tcp[ j ] = (uint8_t) pal[ i ].as<int>(); // index
-                for (size_t c=0; c<3; c++) tcp[j+1+c] = rgbw[c]; // only use RGB component
+                // Apply inverse gamma correction to match built-in palettes
+                uint32_t color = NeoGammaWLEDMethod::inverseGamma32(RGBW32(rgbw[0], rgbw[1], rgbw[2], 0));
+                tcp[j+1] = R(color);
```

Comment:
not applying gamma correction should be
a) a user option, however, this will need a new flag added to the palette.json format
b) a user option, managed by the palette editor - so the palette.json already has gamma correction applied => in this case, not performing any gamma correction in ``loadCustomPalettes()`` would be right